packages/terra-status-view/package.json

```diff
@@ -34,15 +34,13 @@
   "peerDependencies": {
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "terra-base": "^3.7.0",
-    "terra-mixins": "^1.15.0"
+    "terra-base": "^3.11.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "lodash.uniqueid": "^4.0.1",
     "prop-types": "^15.5.8",
     "react-intl": "^2.3.0",
-    "terra-base": "^3.11.0",
```

Comment:
This needs to stay as a dependency. A file from terra-base is imported in the src code, StatusView.jsx

Reply:
We shouldn't have both a module declared as both a dependency and a peer dependency. With terra-base set as a peer dependency, terra-status view will depend on the module consuming it to provide the terra-base include. To pull in terra-base for local testing in this module we can include terra-base as a dev dependency. This won't have the potential to cause a conflict for consumers because dev dependencies won't be installed when the terra-status view is installed as a dependency.

Reply:
Issues logged: 

**Repos**
- https://github.com/cerner/terra-core/issues/1587
- https://github.com/cerner/terra-clinical/issues/new
- https://github.com/cerner/terra-framework/issues/200

**Generator**
- https://github.com/cerner/generator-terra-module/issues/97

**Webpack Config**
- https://github.com/cerner/terra-dev-site/issues/83